### PR TITLE
make modules demo testbooks useable with common server-config

### PR DESCRIPTION
**Pull request by mdellweg**
* canonicalize filenames
* transfer server-configuration into variables
* describe test-runs

### CODE REVIEW

**Comment by mdellweg** [comment]
I know, this is a rather big one, but it is confined to the test directory. And using the common server config, i could run most of the playbooks against my forklift dev-box.
So i would call this a first step towards a useable testsuite.

File: README.md [hunk 1]
```diff
@@ -37,7 +37,27 @@ foreman-ansible-module `modules` and `module_utils` if you do so.
 Now your playbooks and roles should have access to the modules and module_utils
 contained in the repository for use, testing, or development of new modules.
 
-## How to debug this repository
+## How to test modules in this repository
+
+In `test/demo` there are example playbooks for most of the modules.
+To use them, you need a running instance of `foreman` probably with `katello` (use forklift if unsure).
```
Comment (akofink) [inline README.md]: Let's link to forklift here.

**Comment by akofink** [comment]
Towards https://github.com/theforeman/foreman-ansible-modules/issues/61

**Comment by mdellweg** [comment]
@akofink 🏓

**Pull request merged**